Bindings/Java/Matlab/tests/testOsimList2MatlabCell.m

```diff
@@ -0,0 +1,33 @@
+%% clear working space
+clear all;close all;clc;
+
+%% Import OpenSim Libraries
+import org.opensim.modeling.*
+
+%% Instantiate a model from file
+model = Model('../../../../OpenSim/Tests/shared/arm26.osim');
```

Comment:
Because arm26.osim is copied into the test directory, you can simply use `'arm26.osim'` here. We don't want to read files from the repository in the test case.